std/datetime/mod.ts

```diff
@@ -8,6 +8,17 @@ export const MINUTE = SECOND * 60;
 export const HOUR = MINUTE * 60;
 export const DAY = HOUR * 24;
 export const WEEK = DAY * 7;
+const DAYS_PER_WEEK = 7;
+
+enum Day {
+  Sun,
+  Mon,
+  Tue,
+  Wed,
+  Thu,
+  Fri,
+  Sat,
+}
```

Comment:
Should these be exported, or kept private unless/until needed? I didn't see any guidelines about conditions for exporting, keeping exports to a minimal, etc.

Reply:
I suggested to remove the ```currentDayOfYear``` in [that issue](https://github.com/denoland/deno/issues/6601).

As for the purpose of ```std/datetime```: I think a clear definition has to be worked out still, but I think it is a good addition as a std library. In my view the module has theses purposes at the moment:
- provide commonly used functions related to date and time
- provide datetime parser and formatter

I think some 3rd party libraries do the one, some the other.
I did a PR to add a generic parser and a formatter.  In a sense some features are reinventing the wheel (like parser and formatter) but the attention lies to provide compatibility with ```Intl```.
Also this module will likely change once this [proposal](https://github.com/tc39/proposal-temporal) is at stage 4.

Reply:
`Temporal` looks like a step forward. 🧗 

@bartlomieju Any feedback about exports?